central/notifiers/microsoftsentinel/data/example-alert.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
What's the purpose of this file?

Reply:
+1, I'd like to understand that as well

Reply:
Could it be used in some test, may be?

Reply:
I would not like to invest more time into this, if there is a disagreement and this is not seen as useful I can remove this file.

Reply:
My comment was satisfied.